README.md

```diff
@@ -317,34 +400,53 @@ $ docker run -v /mnt/workspace:/mnt/workspace \
 $ # Verify the evidences
 $ docker run -v /mnt/workspace:/mnt/workspace \
              -v /var/run/docker.sock:/var/run/docker.sock \
-             mse-home verify --package package_example_1688025211482089576.tar \
+             mse home verify --package package_example_1688025211482089576.tar \
                              --evidence evidence.json \
                              --output .
 $ # Seal the secrets
 $ docker run -v /mnt/workspace:/mnt/workspace \
              -v /var/run/docker.sock:/var/run/docker.sock \
-             mse-home seal --secrets example/secrets_to_seal.json \
+             mse home seal --secrets example/secrets_to_seal.json \
                            --cert ratls.pem \
                            --output  .
 $ # Complete the configuration
 $ docker run -v /mnt/workspace:/mnt/workspace \
              -v /var/run/docker.sock:/var/run/docker.sock \
             --network=host \
-            mse-home run --sealed-secrets secrets_to_seal.json.sealed test_docker
+            mse home run --sealed-secrets secrets_to_seal.json.sealed test_docker
 $ # Query it
 $ curl https://localhost:7779 --cacert /mnt/workspace/ratls.pem 
 $ # Test it
 $ docker run -v /mnt/workspace:/mnt/workspace \
              -v /var/run/docker.sock:/var/run/docker.sock \
              --network=host \
-             mse-home test --test example/tests \
+             mse home test --test example/tests \
                            --config mse.toml test_docker
 $ # Remove it
 $ docker run -v /var/run/docker.sock:/var/run/docker.sock \
-             mse-home stop --remove test_docker
+             mse home stop --remove test_docker
 ```
 
-The current directory inside the docker is `/mnt/workspace`. You can retrieve all generated files in your current host in: `/mnt/workspace`. Make sure to create it before all at the exact location `/mnt/workspace` (it will not work if both locations are not aligned).  
-
-
-Note: the docker does not contain `mse cloud`. It makes no sense since `mse cloud` is designed to be used with Internet. Plus: you need to login through the web browser, which is not possible using the docker. 
\ No newline at end of file
+## Use case
+
+Let's assume your microservice have to interface with a frontend. The main issue you will face up is that the ratls certificate is not allowed in your web browser. Mainly because it's a self signed cert. Also the ratls extension is not checked by your webbrowser, yet any query to the webservice must verify the ratls extension of the certificate: the security is based on this verification. 
```

Comment:
```suggestion
Let's assume your microservice have to interface with a frontend. The main issue you will face up is that the RA-TLS certificate is not allowed in your web browser, mainly because it's a self signed cert. Also the RA-TLS extension is not checked by your webbrowser, yet any query to the webservice must verify the RA-TLS extension of the certificate: the security is based on this verification. 
```